### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by MichaelEbert**
Don't load non ESS files.
Some changes from meemaw's code that was bugging me
minor style changes

### CODE REVIEW

File: js/cellGenerator.mjs [hunk 1]
```diff
@@ -223,6 +224,28 @@ function initSingleCell(cell, extraColumnName, format = CELL_FORMAT_CHECKLIST){
...
     rowhtml.setAttribute("clid",usableId);
 
+    //img before name
+    if(format & CELL_FORMAT_SHOW_ICON){
+        let htmlIcon;
+        if(!COPYING){
+            htmlIcon = document.createElement("img");
+            htmlIcon.classList.add("itemIcon");
+            htmlIcon.loading = "lazy";
+            htmlIcon.draggable = "false";
```
Comment (MeemawHustlin) [inline js/cellGenerator.mjs:234]: Checklist icons appear to be draggable for some reason even though they are set to not be.

**Review by MeemawHustlin — APPROVED** [review]
checklist icons appear to be draggable for some reason even though they are set to not be, other than that everything looks good. We can probably just fix that small issue later as the main problem was just that dragging these originally would wipe progress, which is no longer an issue.

**Comment by MichaelEbert** [comment]
dragging these originally would wipe progress
oh lol

**Review by MeemawHustlin — APPROVED** [review]
lgtm

**Pull request merged**